uint8 id
float32[3] pos
float32 meas
float32 distortion
bool isDistorted
